tests/unit_tests/external/test_rosco_controller.cpp

```diff
@@ -0,0 +1,9 @@
+#include <Kokkos_Core.hpp>
+#include <gtest/gtest.h>
+
+namespace openturbine::tests {
+
+TEST(ROSCO_Controller, initialize) {
+}
```

Comment:
Are you planning to add anything here for this PR?

Reply:
Yes, added in more recent commit